migrations/env.py

```diff
@@ -90,8 +90,14 @@ def run_migrations_offline():
     script output.
 
     """
+    def include_object(object, name, type_, reflected, compare_to):
+        return not (type_ == "index" or type_== "table" and reflected and compare_to is None)
```

Comment:
The wording is just a bit odd. "include object" but the return is built from exclusions. The logic should be flipped. 

Also, just so we're on the same page you should let us know what the point of this thing is. That logic is not easy to follow. As written, this would require `type_` to be `"table"`, `reflected` would need to be `True`, and `compare_to` would need to be `None`. OR `type_` would need to be `"index"`. But then it returns the opposite of that. 

That's really complicated. Add some parenthesis and flip the logic.